complete/src/main/java/com/example/restservice/dao/Token.java

```diff
@@ -25,6 +20,9 @@
   @GenericGenerator(name = "uuid2", strategy = "uuid2")
   String tokenId;
 
+  @Column(columnDefinition = "integer auto_increment")
```

Comment:
Can we make it start from 1 for each queue? 

When someone creates a new queue, they should start issuing tokens 1, 2, 3.. 

We would have to get the number of tokens issued till now, and add one to it? If this is the route, then we have to take care of atomicity. Otherwise two people might get the same token.

cc: @thehamzarocks